products/models.py

```diff
@@ -55,8 +68,33 @@ class Product(AuditTimeStamp):
     description = models.CharField(
         _("Product Description"), max_length=1000, blank=True
     )
+
+    image = models.ImageField(upload_to=image_path, default="default_image.png")
     name = models.CharField(_("Product Name"), max_length=100)
 
+    def get_price(self):
+        """
+        Returns price with currency
+        Returns:
```

Comment:
Probably make the above a litttle verbose?

Reply:
Done